catchup/catchpointService.go

```diff
@@ -335,12 +339,27 @@ func (cs *CatchpointCatchupService) processStageLedgerDownload() (err error) {
 	return nil
 }
 
-// updateVerifiedAccounts update the user's statistics for the given verified accounts
-func (cs *CatchpointCatchupService) updateVerifiedAccounts(addedTrieHashes uint64) {
+// updateVerifiedCounts update the user's statistics for the given verified hashes
+func (cs *CatchpointCatchupService) updateVerifiedCounts(hashes [][]byte) {
 	cs.statsMu.Lock()
 	defer cs.statsMu.Unlock()
+
+	addedTrieAccountHashes := uint64(0)
+	addedTrieKVHashes := uint64(0)
+	for _, hash := range hashes {
+		if (hash[4] == 3) { // KV
+			addedTrieKVHashes++
+		} else {
+			addedTrieAccountHashes++
+		}
+	}
+
```

Comment:
```suggestion
// Estimate VerfiedAccounts based on the portion of account related hashes (account hashes + resource hashes) we have seen.
```

Reply:
Resolving - I brought in a similar (at least in my mind) comment.  Feel welcomed to make additional suggestions.